src/discordcr/client.cr

```diff
@@ -369,6 +392,7 @@ module Discord
 
     private def handle_reconnect
       # Close the websocket - the reconnection logic will kick in
+      @instant_reconnect = true
```

Comment:
Maybe `@falloff = 0.0`, then add `@falloff = 1.0 if @falloff < 1.0` in `wait_for_reconnect`